mlabs/test/Test/NFT/Init.hs

```diff
@@ -102,12 +102,20 @@ w2 = walletFromNumber 2
 w3 = walletFromNumber 3
 wA = walletFromNumber 4 -- Admin Wallet
 
+{- it was 2 before, but after switching
```

Comment:
I think it's because minting one shot currency requires more time now. It was one TX before, now it's two - first to create utxo at script address, then second to mint.

Reply:
Could we make the contract create the `lookups` and `constraints` without submitting the transaction and then add them to the `Init` Contract to all get submitted in 1 transaction? It could be beneficial if it all happened in one Block/Transaction.